python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -182,7 +190,8 @@ traffic-policies
             log
 {%                             endif %}
 {%                         endif %}
-{%                     elif version is arista.avd.defined('ipv6') %}
+{%                     endif %}
+{%                     if version is arista.avd.defined('ipv6') %}
```

Comment:
what was the issue with elif?

Reply:
![Screenshot 2024-08-02 at 11 51 47 AM](https://github.com/user-attachments/assets/1c012001-3b02-47b8-91c0-532f94d9b03d)
in coverage report it says that condition always satisfied and do not return back to loop condtion